mp2-v1/src/values_extraction/gadgets/column_info.rs

```diff
@@ -69,7 +69,7 @@ impl ColumnInfo {
             slot_input.slot,
             identifier,
             slot_input.byte_offset,
-            slot_input.bit_offset,
+            0, // bit_offset
```

Comment:
Can we add a ToDo to remove this also from internal data structures and from the circuit?